lib/engine/stock_market.rb

```diff
@@ -19,6 +19,7 @@ def initialize(market, unlimited_colors, multiple_buy_colors: [])
           price
         end
       end
+      @par_prices.sort_by!(&:price).reverse!
```

Comment:
why is this being sorted? makes sense but, it's not enough to sort by price, you also need to sort by the indicies since two par prices can have the same value

Reply:
If I choose 18AL from master, when I buy the presidency share from market it will look like this:

Par Price:
$60   $75   $70    $105    $90

After sort it instead looks like:
$60   $70   $75   $90   $105

But I have now added so it is sorted by price, column, row - reversed.